cmd/tetra/debug/stats.go

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package debug
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/cilium/ebpf"
+	"github.com/spf13/cobra"
+	"golang.org/x/sys/unix"
+)
+
+func NewEnableStatsCmd() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "enable-stats",
+		Short: "Enable BPF stats",
+		Run: func(_ *cobra.Command, _ []string) {
+			// We enable BPF stats system wide keep it enabled as long as
+			// the stats descriptor is open - app is running
+			_, err := ebpf.EnableStats(uint32(unix.BPF_STATS_RUN_TIME))
+			if err != nil {
+				log.Fatal(fmt.Errorf("failed to enable stats: %v", err))
```

Comment:
nit: you could use `RunE func(cmd *Command, args []string) error` and return an error here directly so that `tetra` terminates with error code != 0 and prints the error directly.

btw cool that you put that cmd under `debug` I was going to suggest it before reading it :)!

Reply:
> nit: you could use `RunE func(cmd *Command, args []string) error` and return an error here directly so that `tetra` terminates with error code != 0 and prints the error directly.

ok, will use that one